container/container_unit_test.go

```diff
@@ -66,3 +70,52 @@ func TestContainerSecretReferenceDestTarget(t *testing.T) {
 		t.Fatalf("expected secret dest %q; received %q", expected, d)
 	}
 }
+
+func TestContainerLogPathSetForJSONFileLogger(t *testing.T) {
+	containerRoot, err := ioutil.TempDir("", "TestContainerLogPathSetForJSONFileLogger")
```

Comment:
Oh sorry... premature merge...
This needs to be cleaned up in a defer.
```go
defer os.RemoveAll(containerRoot)
```

Can you open a PR to fix that?
Thanks!

Reply:
Opened a cleanup PR at https://github.com/moby/moby/pull/36429